src/internal/TouchFreeEvents/TouchFreeEvents.ts

```diff
@@ -22,9 +22,10 @@ export interface TouchFreeEventSignatures {
      */
     onTrackingServiceStateChange: (state: TrackingServiceState) => void;
     /**
-     * Event dispatched when the status of TouchFree Service changes
+     * Event dispatched when the status of TouchFree Service changes.
+     * An event for 'Disconnected' will be sent only if it was connected before.
```

Comment:
🤷An event for 'Disconnected' will only be sent if a connection was already established